cores/cv32e40p/user_manual/source/debug.rst

```diff
@@ -45,21 +45,31 @@ Interface
 +===============================+===========+============================================+
 | ``debug_req_i``               | input     | Request to enter Debug Mode                |
 +-------------------------------+-----------+--------------------------------------------+
+| ``debug_havereset_o``         | output    | Debug status: Core has been reset          |
++-------------------------------+-----------+--------------------------------------------+
+| ``debug_running_o``           | output    | Debug status: Core is running              |
++-------------------------------+-----------+--------------------------------------------+
+| ``debug_halted_o``            | output    | Debug status: Core is halted               |
++-------------------------------+-----------+--------------------------------------------+
 | ``dm_halt_addr_i[31:0]``      | input     | Address for debugger entry                 |
 +-------------------------------+-----------+--------------------------------------------+
 | ``dm_exception_addr_i[31:0]`` | input     | Address for debugger exception entry       |
 +-------------------------------+-----------+--------------------------------------------+
 
 ``debug_req_i`` is the "debug interrupt", issued by the debug module when the core should enter Debug Mode. The ``debug_req_i`` is synchronous to ``clk_i`` and requires a minimum assertion of one clock period to enter Debug Mode. The instruction being decoded during the same cycle that ``debug_req_i`` is first asserted shall not be executed before entering Debug Mode.
 
```

Comment:
Consider adding the following:
`debug_havereset_o` , `debug_running_o`, and `debug_mode_o` signals provide the operational status of the core to the debug module. The assertion of these signals are mutually exclusive.

Reply:
done

Reply:
Done